modules/ROOT/pages/procedures.adoc

```diff
@@ -372,30 +374,6 @@ m|DBMS
 | *Required provileges* 3+| `SERVER MANAGEMENT`
 |===
 
-[role=label--enterprise-edition label--new-5.23]
```

Comment:
Why do you remove this procedure?

Reply:
I can see it in the code https://github.com/neo-technology/neo4j/blob/5.26/private/enterprise/neo4j-dbms/src/main/java/com/neo4j/dbms/systemgraph/TopologyGraphProcedures.java#L214

Or I'm missing something?

Reply:
Oh, sorry, I thought it duplicated `dbms.cluster.deallocateDatabaseFromServer`.

Reply:
There is an odd failure, which I was trying to solve. I'll put it back.